bigbluebutton-html5/imports/ui/components/app/component.jsx

```diff
@@ -158,16 +158,33 @@ class App extends Component {
   renderUserListResizable() {
     const { userList } = this.props;
 
+    // Variables for resizing user-list.
+    const USERLIST_MIN_WIDTH_PX = 100;
+    const USERLIST_MAX_WIDTH_PX = 240;
+    const USERLIST_DEFAULT_WIDTH_RELATIVE = 18;
+
+    // prevent the default width size of the userList to exceed the maxWidth
+    const USERLIST_DEFAULT_WIDTH = (window.innerWidth * (USERLIST_DEFAULT_WIDTH_RELATIVE / 100.0)) < USERLIST_MAX_WIDTH_PX ? `${USERLIST_DEFAULT_WIDTH_RELATIVE}%` : USERLIST_MAX_WIDTH_PX;
+
     if (!userList) return null;
 
     return (
       <Resizable
-        minWidth="10%"
-        maxWidth="20%"
+        defaultSize={{ width: USERLIST_DEFAULT_WIDTH }}
+        minWidth={USERLIST_MIN_WIDTH_PX}
+        maxWidth={USERLIST_MAX_WIDTH_PX}
         ref={(node) => { this.resizableUserList = node; }}
         className={styles.resizableUserList}
-        enable={{ top: false, right: true, bottom: false, left: false, topRight: false,
-          bottomRight: false, bottomLeft: false, topLeft: false }}
+        enable={{
+ top: false,
+right: true,
+bottom: false,
+left: false,
+topRight: false,
+          bottomRight: false,
+bottomLeft: false,
+topLeft: false,
+}}
```

Comment:
Could you fix these indentations?

Reply:
Ok

Reply:
Odd. I see a commit where the indentations are fixed but github's view is not updated. Is this PR good to be merged?

Reply:
Weird. I have checked my branch and the last change what I’ve commited was not applied on the code of the branch..

I will check it thanks!